socialhome/static/sass/streams.scss

```diff
@@ -1,7 +1,8 @@
 @import "variables";
 
 /* New content container */
-#new-content-container {
+#new-content-container,
+.new-content-container {
```

Comment:
These duplications in this file are unnecessary as other comment re class vs id.